relay/nakama/Dockerfile

```
@@ -3,18 +3,20 @@ FROM heroiclabs/nakama-pluginbuilder:3.22.0 AS builder
 
-WORKDIR /nakama
+WORKDIR /nakama/plugin
+
+COPY relay/nakama .
+COPY sign /sign
 
-COPY go.mod go.sum ./
 RUN go mod download
 
-COPY . .
 RUN go build --trimpath --buildmode=plugin -o ./plugin.so
```

Hadolint

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)